examples/env_browser.js

```diff
@@ -1,16 +1,23 @@
 // eslint-disable-next-line no-undef
+/** 
+ * The following configuration is used in the browser examples
+ * (index.html and giraffe.html).
+ *
+ * Replace the values with your own InfluxDB values.
+ */
```

Comment:
This change is required to appease linter, the CI tests also control the style and format of examples.

```suggestion
/*
 * The following configuration is used in the browser examples
 * (index.html and giraffe.html).
 *
 * Replace the values with your own InfluxDB values.
 */
// eslint-disable-next-line no-undef
```